public/app/plugins/panel/barchart/utils.ts

```diff
@@ -301,7 +301,7 @@ export function prepareBarChartDisplayValues(
   }
 
   // Bar chart requires a single frame
-  const frame = series.length === 1 ? series[0] : outerJoinDataFrames({ frames: series, enforceSort: false });
```

Comment:
this is now handled automagically by skipping sorting of string fields. (we still want to sort BarCharts by time x)

Reply:
barchart... I don't think so.  candlestick maybe